minimal-launchpad/minimal_ui_index.js

```diff
@@ -300,7 +300,11 @@ consoleStartButton.onclick = async () => {
     if (config.portConnectionOptions?.length) {
         await transport.connect(parseInt(config.portConnectionOptions[0]?.baudRate), serialOptions);
     } else {
-        await transport.connect();
+        if (esploader.chip.CHIP_NAME === "ESP32-C2") {
```

Comment:
Same comment as above, checking chip frequency rather than chip name

Reply:
Implement the suggested changes.